SPECS/kubernetes/kubernetes-1.16.13.spec

```diff
@@ -0,0 +1,175 @@
+%global debug_package %{nil}
+%ifarch x86_64
+%define archname amd64
+%endif
+%ifarch aarch64
+%define archname arm64
+%endif
+Summary:        Microsoft Kubernetes
+Name:           kubernetes
+Version:        1.16.13
```

Comment:
we can skip on the 1.16 versions because it will be dropped as soon as 1.19 is available.

Reply:
removed 1.16 spec ...